### PR TITLE
Small text cleanups for README and charmcraft help

### PR DESCRIPTION
**Pull request by markshuttle**
Lots of text work, I tried not to change anything that might break tests but the occasional %s might have slipped through :)

### CODE REVIEW

File: charmcraft/commands/store/__init__.py [hunk 1]
```diff
@@ -52,56 +52,61 @@ def get_name_from_metadata():
 
 
 class LoginCommand(BaseCommand):
-    """Log into the store."""
+    """Login to Charmhub."""
```
Comment (chipaca) [inline charmcraft/commands/store/__init__.py:55]: AIUI the verb is "log in", and "login" the noun, so this (and a few other below) should be
```suggestion
    """Log in to Charmhub."""
```
(not "into" as the emphasis is on the "in")

**Review by chipaca — APPROVED** [review]
+1 but I think 'login' vs 'log in' needs fixing (can be in a follow up; this is already big)

File: charmcraft/commands/store/__init__.py (hunk 1 above)
Comment (markshuttle) [reply charmcraft/commands/store/__init__.py:55]: Srsly? :)

**Review by cmatsuoka — COMMENTED** [review]
Thanks. The text is very consistent, clear and informative. I just added a note about a typo in the universal operators URL.

File: README.md [hunk 2]
```diff
@@ -1,60 +1,145 @@
...
+You can use charmcraft with operators written in any language but we
+recommend the [Python Operator Framework on
+Github](https://github.com/canonical/operator) which is also [on
+PyPI](https://pypi.org/project/ops/) for ease of development and
+collaboration.
 
-## How do I install it?
+Charmcraft and the Python Operator Framework extend the operator pattern
+beyond Kubernetes with [universal
+operators](https://juju.is/universdal-operators) that drive Linux and
```
[the remark this answers is not in the document]
Comment (markshuttle) [reply README.md:25]: OK, I'll address that

[the remark this answers is not in the document]
Comment (markshuttle) [reply README.md:25]: Actually, I seem to have hit the limits of my git-fu, so please could you address that on landing?

[the remark this answers is not in the document]
Comment (facundobatista) [reply README.md]: Great catch! Fixed

**Review by facundobatista — APPROVED** [review]

**Pull request merged**